oneview/resource_server_profile_template.go

```diff
@@ -1827,7 +1824,6 @@ func resourceServerProfileTemplateRead(d *schema.ResourceData, meta interface{})
 	}
 	d.Set("etag", spt.ETAG)
 	d.Set("hide_unused_flex_nics", spt.HideUnusedFlexNics)
-	d.Set("initial_scope_uris", spt.InitialScopeUris)
```

Comment:
i can under volume also we have initial_scope_uris, not sure if we handle that as well. But there is one thing: if volume created as part of SP then initial_scope_uris should not be provided

Reply:
Not fixing it as it is part of san storage and we have not looked into it.